graphics/macgui/macwindow.cpp

```diff
@@ -263,6 +263,8 @@ void MacWindow::updateOuterDims() {
 }
 
 void MacWindow::drawBorder() {
+	resizeBorderSurface();
```

Comment:
This will reallocate border surface unconditionally on every change. An example of a change that does not change dimensions is call to `setTitle()`. Thus, please, implement inside of `resizeBorderSurface()` a check that it skips surface reallocation in case border dimensions are not changed